packages/spotlight/src/container.js

```diff
@@ -948,6 +951,82 @@ function isWithinOverflowContainer (target, containerIds = getContainersForNode(
 		.some(config => config && config.overflow);
 }
 
+/**
+ * Notifies any affected containers that focus has left one of their children for another container
+ *
+ * @param {String} direction up/down/left/right
+ * @param {Node} current currently focused element
+ * @param {String[]} currentContainerIds Containers for current
+ * @param {Node} next To be focused element
+ * @param {String[]} nextContainerIds Containers for next
+ * @private
+ */
+function notifyBlurContainer (direction, current, currentContainerIds, next, nextContainerIds) {
+	currentContainerIds.forEach(containerId => {
+		if (!nextContainerIds.includes(containerId)) {
+			const config = getContainerConfig(containerId);
+
+			if (config && config.onBlurContainer) {
+				config.onBlurContainer({
+					type: 'onBlurContainer',
+					direction,
+					target: current,
+					relatedTarget: next
+				});
+			}
+		}
+	});
+}
+
+/**
+ * Notifies any containers that focus attempted to move but failed to find a target
+ *
+ * @param {String} direction up/down/left/right
+ * @param {Node} current currently focused element
+ * @param {String[]} currentContainerIds Containers for current
+ * @private
+ */
+function notifyBlurFailContainer (direction, current, currentContainerIds) {
```

Comment:
Should we rename this to `notifyBlurContainerFail` in order to match the accompanying `onBlurContainerFail` naming convention used in the config?